src/App.less

```
@@ -381,23 +381,65 @@ input::placeholder {
+  .ant-table-column-sorter-inner {
+    flex-direction: row;
+    align-items: center;
+    gap: 2px;
+  }
+
   .ant-table-column-sorter-up,
   .ant-table-column-sorter-down {
-    transition: all 0.2s;
+    width: 12px;
+    height: 12px;
+    background-color: currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)